modules/nixos/services/bobaboard/default.nix

```diff
@@ -87,20 +114,63 @@ in
         Type = "oneshot";
         User = "bobaboard";
         Group = "bobaboard";
-        ExecStart = "${bobabackend-packages.bobadatabase}/bin/bobadatabase";
+        # ExecStart = "${bobabackend-packages.bobadatabase}/bin/bobadatabase";
         
         # Uncomment this to have the script re-run when boba-server restarts
         # RemainAfterExit = "yes";
       };
 
       # Only run the script if the .migrate file is not present
       # Remove ExecStart when enabling this
-      # script = ''
-      #   if ! [ -f /var/lib/bobaboard/.migrate ]; then
-      #     touch /var/lib/bobaboard/.migrate
-      #     ${bobabackend-packages.bobadatabase}/bin/bobadatabase
-      #   fi
-      # '';
+      script = ''
+        if ! [ -f /var/lib/bobaboard/.migrate ]; then
+          touch /var/lib/bobaboard/.migrate
+          ${bobabackend-packages.bobadatabase}/bin/bobadatabase
+        fi
+      '';
+    };
+
+    security.acme.acceptTerms = true;
+    security.acme.defaults.email = "essential.randomn3ss@gmail.com";
+    services.nginx = {
+      enable = true;
+      recommendedProxySettings = true;
+      recommendedTlsSettings = true;
+      virtualHosts."frontend" =  {
```

Comment:
The name for the virtual hosts should be unique. We'll probably want to do something like `bobaboard-frontend` and `bobaboard-backend` to distinguish them.

Reply:
Alternatively, the default server name for each vhost is the name specified here. That means we can use the domain name instead and it'll work.

We can refactor to this in the future since the goal here is testing to make sure it works, but wanted to mention it nonetheless.